src/account.c

```diff
@@ -183,6 +184,7 @@ handle_get_user_information (XdpDbusAccount *object,
                              const gchar *arg_parent_window,
                              GVariant *arg_options)
 {
+  Account *account = (Account *) object;
```

Comment:
Any reason you're using normal casts instead of checked casts? `ACCOUNT (object)` rather than `Account *object`?

Reply:
I had the same thought, wondering why GBM needs to be involved in GPU copies at all when EGL exists. But I'm not confident it would be a real solution. Still, something nice to try in a separate branch some day.

Reply:
What I meant to say was: maybe we should have a `XDP_DECLARE_PORTAL()` & `XDP_DEFINE_PORTAL()` macros that handle setting up the types, so that we can continue to clearly distinguish between "plumbing" (`Xdp*`) and actual portal implementations.